contrib/go-redis/redis.v7/example_test.go

```diff
@@ -9,39 +9,40 @@ import (
 	"context"
 	"time"
 
-	"github.com/go-redis/redis/v8"
-	redistrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/go-redis/redis.v8"
+	"github.com/go-redis/redis/v7"
+	redistrace "gopkg.in/DataDog/dd-trace-go.v1/contrib/go-redis/redis.v7"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
 	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
 )
 
 // To start tracing Redis, simply create a new client using the library and continue
 // using as you normally would.
 func Example() {
-	ctx := context.Background()
 	// create a new Client
 	opts := &redis.Options{Addr: "127.0.0.1", Password: "", DB: 0}
 	c := redistrace.NewClient(opts)
 
 	// any action emits a span
-	c.Set(ctx, "test_key", "test_value", 0)
+	c.Set("test_key", "test_value", 0)
 
 	// optionally, create a new root span
-	root, ctx := tracer.StartSpanFromContext(context.Background(), "parent.request",
+	root, _ := tracer.StartSpanFromContext(context.Background(), "parent.request",
 		tracer.SpanType(ext.SpanTypeRedis),
 		tracer.ServiceName("web"),
 		tracer.ResourceName("/home"),
 	)
 
+	// set the context on the client
```

Comment:
We shouldn't leave commented out code anywhere.

Reply:
Uncommented it